tox.ini

```
@@ -0,0 +1,79 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+[tox]
+skipsdist=True
+skip_missing_interpreters = True
+envlist = lint, unit
+
+[vars]
+src_path = {toxinidir}/src/
+tst_path = {toxinidir}/tests/
+;lib_path = {toxinidir}/lib/charms/operator_name_with_underscores
+all_path = {[vars]src_path} {[vars]tst_path}
+
+[testenv]
+setenv =
+  PYTHONPATH = {toxinidir}:{toxinidir}/lib:{[vars]src_path}
+  PYTHONBREAKPOINT=ipdb.set_trace
+  PY_COLORS=1
+passenv =
+  PYTHONPATH
+  HOME
+  PATH
+  CHARM_BUILD_DIR
+  MODEL_SETTINGS
+  HTTP_PROXY
+  HTTPS_PROXY
+  NO_PROXY
```

```suggestion
  PYTHONPATH
  HOME
  CHARM_BUILD_DIR
  MODEL_SETTINGS
```

These are passed through automatically I believe

---

You're right, Jon. These variables are passed by default in [Tox](https://github.com/tox-dev/tox/blob/4cf816c97de18dca9d99dc86fec341c1d2c4a20a/src/tox/config/__init__.py#L769).